uccaapp/export_units_by_filter.py

```diff
@@ -21,25 +22,29 @@ def get_top_level_ancestor(node):
             parent = parent.fparent
         return parent
 
-def main(output = None, comment = False, categories = (), **kwargs):
+
+def main(output=None, comment=False, categories=(), **kwargs):
     filtered_nodes = []
     for passage, task_id, user_id in TaskDownloader(**kwargs).download_tasks(**kwargs, write=False):
         for node in passage.layer(layer1.LAYER_ID).all:
             if comment and node.extra.get("remarks"):
-                filtered_nodes.append(("comment",node,task_id,user_id))
+                filtered_nodes.append(("comment", node, task_id, user_id))
             else:
-                all_tags = node.extra.get("all_tags")
+                all_tags = []
```

Comment:
This is fine, but FYI, you could write it as `all_tags = [c.tag for edge in node for c in edge.categories]`